newrelic/api/error_trace.py

```diff
@@ -14,24 +14,25 @@
 
 import functools
 
-from newrelic.api.time_trace import current_trace, record_exception
+from newrelic.api.time_trace import current_trace, notice_error
 from newrelic.common.object_wrapper import FunctionWrapper, wrap_object
 
 
 class ErrorTrace(object):
-
-    def __init__(self, ignore_errors=[], **kwargs):
+    def __init__(self, ignore=None, expected=None, status_code=None, **kwargs):
```

Comment:
Was thinking about this API, can we keep `ignore_errors` in addition to the others?

We should add a deprecation warning to the ignore_errors argument and forward the value to ignore_classes if ignore_classes is Falsey, or we could merge them if they're both iterables. Would be less likely to break existing customer code bases.